tests/data/testdb/README.md

```diff
@@ -0,0 +1,7 @@
+# test database folder
+
+This folder contains two local database instances for unit tests.
+
+The db `message_ix_legacy` should contain a scenario built from the previous
```

Comment:
can you add here how to actually do this for future users? I know it seems obvious, but as soon as we ask someone new to assist, they will ask questions =) 

So basically just a list of commands, what to add in the code, etc.

Reply:
thanks, I added some explanation here.